build_web_compilers/test/dart2wasm_bootstrap_test.dart

```diff
@@ -69,9 +69,11 @@ void main() {
           'a|web/index.dart.js': decodedMatches(
             stringContainsInOrder(
               [
-                'if (supportsWasmGC())',
+                'if',
```

Comment:
Are there more interesting cases to cover now?

And: please mention in a comment where this string is now coming from, i.e. that it's via the `supportExpression` in `Dart2WasmBootstrapResult`.

Reply:
The challenge for the test is that we no longer now that the contents of the `if` statement here are, because they depend on what `dart2wasm` emits. So I had to generalize the test and added a comment explaining the reasons.

Reply:
Thanks.